python-modules/cis_change_service/cis_change_service/profile.py

```diff
@@ -0,0 +1,97 @@
+"""Get the status of an integration from the identity vault and auth0."""
+import json
+from cis_aws import connect
+
+
+class Vault(object):
+    """Handles flushing profiles to Dynamo when running local or in stream bypass mode."""
+    def __init__(self, sequence_number):
+        self.connection_object = connect.AWS()
+        self.identity_vault_client = None
+        self.sequence_number = sequence_number
+
+    def _connect(self):
+        self.connection_object.session()
+        self.connection_object.assume_role()
+        self.identity_vault_client = self.connection_object.identity_vault_client()
+        return self.identity_vault_client
+
+    def put_profile(self, profile_json):
+        """Write profile to the identity vault."""
+        if self.identity_vault_client is None:
+            self._connect()
+
+        user_id = self._get_id(profile_json)
```

Comment:
i wonder if the vault should use `cis_profile` here

Reply:
It should.  I suggest backlogging integrating cis_profile to after we finish the processor modules.

Reply:
Added a note to come back into this later.